### PR TITLE
build: lock file maintenance

### PR DESCRIPTION
**Pull request by alan-agius4**
See associated pull request for more information.

(cherry picked from commit d5f92bdd64a082e42227a9a2d766654bb028c9a9)

### CODE REVIEW

File: packages/core/test/bundling/animations/bundle.golden_symbols.json [hunk 1]
```diff
@@ -218,9 +212,6 @@
   {
     "name": "ENVIRONMENT_INITIALIZER"
   },
-  {
-    "name": "ERROR_ORIGINAL_ERROR"
-  },
```
Comment (devversion) [inline packages/core/test/bundling/animations/bundle.golden_symbols.json:223]: Do we understand why so many symbols are gone here? I assume Terser might have changed something? 

I can help looking into it if you want. Potentially we would need to change the symbol extractor to "understand" a new "format"?

**Review by devversion — APPROVED** [review]
LGTM overall. Just one question on the symbol goldens

File: packages/core/test/bundling/animations/bundle.golden_symbols.json (hunk 1 above)
Comment (devversion) [reply packages/core/test/bundling/animations/bundle.golden_symbols.json:223]: seems reasonable.

**Review by jessicajaniuk — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
Caretaker: This is safe to merge with the current set of reviews.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit cd5946bccda425d33d4ebad58685aca416e9260a.

**Pull request closed without merging**